src/qt/optionsdialog.cpp

```diff
@@ -41,8 +41,17 @@ OptionsDialog::OptionsDialog(QWidget *parent, bool enableWallet) :
     /* Main elements init */
     ui->databaseCache->setMinimum(nMinDbCache);
     ui->databaseCache->setMaximum(nMaxDbCache);
+    static const uint64_t GiB = 1024 * 1024 * 1024;
+    static const uint64_t nMinDiskSpace = MIN_DISK_SPACE_FOR_BLOCK_FILES / GiB +
+                          (MIN_DISK_SPACE_FOR_BLOCK_FILES % GiB) ? 1 : 0;
```

Comment:
I would recommend moving this to `optionsmodel.h` (perhaps also as `constexpr`?) so that we can reuse it when setting a default (see next note)

Reply:
We can remove the expression and just put a 3:
```
ui->pruneSize->setMinimum(3); // Set default to 3 GB (2200MB rounded)
```

Reply:
That would mean that if we would change the value in `validation.h`, we need to *remember* that we need to change this value in multiple places (here and `optionsmodel.cpp`) too, but with my suggestion, it would automatically update it and we can just chill. 

That allows us to be lazy long-term by doing a little more now 😁

Reply:
That makes sense :+1: